samples/bot-tag-mention/csharp/appsettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "MicrosoftAppType": "Multitenant",
+  "MicrosoftAppId": "",
+  "MicrosoftAppPassword": "",
+  "MicrosoftAppTenantId": "",
+  "ConnectionName": ""
```

Comment:
Mention about this connection also in readme file. How to get it.

Reply:
Fixed